addon/utils/cookies.js

```diff
@@ -0,0 +1,31 @@
+let Cookies = {
+  create(name, value, days) {
+    var expires;
+
+    if (days) {
+      var date = new Date();
+      date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
+      expires = "; expires=" + date.toGMTString();
+    } else {
+      expires = "";
+    }
+    document.cookie = encodeURIComponent(name) + "=" + encodeURIComponent(value) + expires + "; path=/";
```

Comment:
A domain is not required for localhost. I will confirm this on staging once we get up and make a followup commit.

Seems like we could need a `cookieDomain` config value.